src/rpc/blockchain.cpp

```diff
@@ -930,7 +937,8 @@ static RPCHelpMan getblock()
     return RPCHelpMan{"getblock",
                 "\nIf verbosity is 0, returns a string that is serialized, hex-encoded data for block 'hash'.\n"
                 "If verbosity is 1, returns an Object with information about block <hash>.\n"
-                "If verbosity is 2, returns an Object with information about block <hash> and information about each transaction. \n",
+                "If verbosity is 2, returns an Object with information about block <hash> and information about each transaction.\n"
+                "If verbosity is 3, returns an Object with information about block <hash> and information about each transaction, including prevout information for inputs (only for blocks in the current best chain).\n",
```

Comment:
FMI: "only for blocks in the current best chain" - Is this precisely worded with regard to how pruning works?

Reply:
iiuc, txundo will not be looked up for pruned blocks. I reworded this from "only for blocks" to "only for unpruned blocks".